.kokoro/build.sh

```diff
@@ -80,11 +80,11 @@ nightly-it)
           verify
     RETURN_CODE=$?
     ;;
-#graalvm)
-#    # Run Unit and Integration Tests with Native Image
-#    mvn -B ${INTEGRATION_TEST_ARGS} -ntp -Pnative -Penable-integration-tests test
-#    RETURN_CODE=$?
-#    ;;
+graalvm)
+    # Run Unit and Integration Tests with Native Image
+    mvn -B ${INTEGRATION_TEST_ARGS} -ntp -Pnative -Penable-integration-tests test
+    RETURN_CODE=$?
+    ;;
```

Comment:
could you also remove the `build.sh` exclusion from `owlbot.py`?

Reply:
good catch, done!